dashboard/templatetags/cards.py

```diff
@@ -22,6 +22,9 @@ def card_diaperchange_last(child):
     """
     instance = models.DiaperChange.objects.filter(
         child=child).order_by('-time').first()
+
+    if not instance and getattr(Settings.objects.first(), 'dashboard_hide_empty'):
+        return {'hidden': True}
```

Comment:
@jcgoette I think you missed a step here. You are assigning this variable throughout but then not using it anywhere? You probably want to reference in to `dashboard/templates/cards/base.html` to decide whether or not to render the card.